deploy/docker/seqr/Dockerfile

```diff
@@ -12,22 +12,37 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     htop \
     less \
     nano \
-    wget \
     xterm \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-
-# install python
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    python-dev \
-    ca-certificates \
-    && wget https://bootstrap.pypa.io/get-pip.py \
-    && python get-pip.py \
-    && pip install --upgrade pip setuptools \
+#  install the packages necessary to build Python source:
+RUN apt update && apt install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev \
+    libnss3-dev libssl-dev libreadline-dev libffi-dev libbz2-dev \
+    libncursesw5-dev libsqlite3-dev tk-dev libc6-dev ca-certificates \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+ARG PYTHON_MAIN_VER="3.7"
+ARG PYTHON_MINOR_VER="8"
+ARG PYTHON_VERSION=$PYTHON_MAIN_VER.$PYTHON_MINOR_VER
+
+# downland and install Python from soruce code
+WORKDIR /
+RUN curl -O https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tar.xz \
+    && tar -xf Python-$PYTHON_VERSION.tar.xz \
+    && rm -f Python-$PYTHON_VERSION.tar.xz
+
+WORKDIR /Python-$PYTHON_VERSION
+RUN ./configure --enable-optimizations \
```

Comment:
Codacy found an issue: [Quote this to prevent word splitting.](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5815111)